apps/vr-tests-web-components/package.json

```diff
@@ -21,7 +21,7 @@
     "typescript": "4.7.4"
   },
   "dependencies": {
-    "@fluentui/react-button": "^9.2.3",
+    "@fluentui/react-button": "*",
     "@fluentui/react-storybook-addon": "*",
     "@fluentui/web-components": "3.0.0-beta.15",
```

Comment:
this is invalid and should be `>=3.0.0-alpha`, will be normalized once this lands to master https://github.com/microsoft/fluentui/pull/31293#pullrequestreview-2042600850